chkdm

```diff
@@ -64,32 +64,36 @@ declare -A nofilterDNS secureDNS adblockDNS
 nofilterDNS[AdGuard]="94.140.14.140"
 nofilterDNS[Cloudflare]="1.1.1.1"
 nofilterDNS[dns0.eu]="193.110.81.254"
-nofilterDNS[Freenom World]="80.80.80.80"
+nofilterDNS[Freenom World]="80.80.81.81" ### BUG: 80.80.80.80 timeout in DE-only
 nofilterDNS[Google]="8.8.8.8"
 nofilterDNS[Hinet]="168.95.1.1"
-nofilterDNS[UltraDNS]="64.6.64.6"
 nofilterDNS[OpenDNS]="208.67.222.2"
 nofilterDNS[Quad9]="9.9.9.10"
+nofilterDNS[UltraDNS]="64.6.64.6"
 nofilterDNS[Yandex]="77.88.8.1"
 
+secureDNS[CONTROL D]="76.76.2.1"
 secureDNS[CleanBrowsing]="185.228.168.9"
 secureDNS[Cloudflare]="1.1.1.2"
 secureDNS[Comodo]="8.26.56.26"
-secureDNS[CONTROL D]="76.76.2.1"
 secureDNS[dns0.eu]="193.110.81.0"
-secureDNS[UltraDNS]="156.154.70.2"
-secureDNS[SafeDNS]="195.46.39.39"
 secureDNS[OpenDNS]="208.67.222.222"
 secureDNS[Quad101]="101.101.101.101"
 secureDNS[Quad9]="9.9.9.9"
+secureDNS[SafeDNS]="195.46.39.39"
+secureDNS[UltraDNS]="156.154.70.2"
 secureDNS[Yandex]="77.88.8.2"
 
 adblockDNS[AdGuard]="94.140.14.14"
 adblockDNS[AhaDNS]="5.2.75.75"
+adblockDNS[clean.dnsforge.de]="49.12.223.2"
 adblockDNS[CONTROL D]="76.76.2.2"
-adblockDNS[dnsforge.de]="176.9.93.198"
 adblockDNS[OVPN]="192.165.9.157"
-adblockDNS[Tiarap]="188.166.206.224"
+#adblockDNS[Tiarap]="188.166.206.224" ### BUG: timeout in EU
+
+# Not following: chkdm.additional_dns was not specified as input (see shellcheck -x). [SC1091]
+# shellcheck disable=SC1091
+[ -f chkdm.additional_dns ] && . chkdm.additional_dns
```

Comment:
Not a good idea just to directly run an additional script, if you want to use extra resolver list, shouldn't it just be a list to be read?

Reply:
still not a good idea to source an external just like that 😅